ice_setup/ice.py

```diff
@@ -1334,6 +1498,7 @@ def default(package_path, use_gpg):
         '2. Install Calamari web application on the ICE Node (current host)',
         '3. Install ceph-deploy on the ICE Node (current host)',
         '4. Configure host as a ceph and calamari minion repository for remote hosts',
+        '5. Calamari minion repository setup',
```

Comment:
this reads odd to me, in step 4 it says it is configuring the hose as a ceph and calamari minion repository, but here it says something similar. 

What is the difference? (maybe I added this? :/ )